src/NuGet.Clients/VsExtension/VisualStudioCredentialProvider.cs

```diff
@@ -4,11 +4,11 @@
 using System;
 using System.Net;
 using System.Threading;
-using System.Threading.Tasks;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
 using NuGet.Configuration;
 using NuGet.Credentials;
+using Tasks = System.Threading.Tasks;
```

Comment:
Is this actually needed? With `using System.Threading` Tasks.Task can be resolved just fine.